python_modules/dagster/dagster/_cli/code_server.py

```diff
@@ -130,7 +130,7 @@ def code_server_cli():
     "--startup-timeout",
     type=click.INT,
     required=False,
-    default=180,
+    default=0,
```

Comment:
theres also a "show_default" (or something close to that) you can use to have the default value shown

Reply:
yeah fair enough - in this case the default is somewhat ambiguous, see first bullet of test plan